### ISSUE TITLE
Bring Back Tripping After Jumping 

### ISSUE THREAD

**Issue by MamaPrince** [echoroleplay/feedback#361]
**Suggestion Description**
Tripping after jumping seems to have been removed, but we really need this abck because watching streams people are bunny hopping to powergame moving faster

**Reason**
Stated above, nothing else needed

**Additional context**
N/A

**Comment by Lane2G** [comment]
Yeah, no thanks 💀 it caused way more harm then good. it’s easier just to report the ones abusing it then revert back to the clunky old system. the amount of times it has ruined chases and gunfights for the entire server was reason enough for it to be removed.

**Comment by MamaPrince** [comment]
I never had an issue with it so I wouldn’t understand where your coming from

**Comment by karichandesu** [comment]
Same, never had an issue with it either.

**Comment by claymstehling** [comment]
Oh I have, I like the no stumbling now and just reporting the people that abuse it
